ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Force UTC time
+export TZ=UTC
```

Comment:
Just checking, but I assume that we verified that changing the timezone won't cause any problems for the peering partners?